homework/index.js

```diff
@@ -0,0 +1,22 @@
+// Get the drop down list
```

Comment:
Very nice code again! Also nicely consistent with the comments (capitals, no full stops - I like this!). However, the code speaks for itself, so you can probably drop the comments :).

Nice variable naming too!